picard/ui/options/plugins.py

```diff
@@ -246,30 +262,35 @@ def add_plugin_item(self, plugin, item=None):
         else:
             item.setText(1, plugin.version)
 
-        label = None
         if plugin.can_be_updated:
-            label = _("Update")
+            self.ui.plugins.headerItem().setText(3, _("Update"))
+            button = QtWidgets.QPushButton(_("Update"))
+            button.setMaximumHeight(button.fontMetrics().boundingRect(_("Update")).height() + 7)
```

Comment:
Please store `_("Update")` in a local variable. `label = _("Update")` and replace 3 occurences with `label`.